src/lib/holocene/checkbox.svelte

```diff
@@ -137,8 +143,12 @@
     @apply outline outline-interactive;
   }
 
+  input:checked ~ .checkmark {
+    @apply border-interactive bg-interactive;
+  }
+
   .checkmark {
-    @apply relative box-content flex h-4 w-4 flex-none cursor-pointer rounded-sm border border-primary bg-white dark:bg-transparent;
+    @apply relative box-content flex h-4 w-4 flex-none cursor-pointer rounded-md border-2 border-subtle bg-transparent text-white hover:border-white/30 hover:bg-interactive  hover:shadow-[inset_0_0_0_1.5px] hover:shadow-white hover:dark:border-black/30 hover:dark:shadow-black;
```

Comment:
I think we can make this match the designs a little better by utilizing `outline` ⬇️ 

```suggestion
    @apply relative box-content flex h-4 w-4 flex-none cursor-pointer rounded-md border-2 border-subtle bg-transparent text-white outline outline-4 outline-transparent hover:border-transparent hover:border-white hover:bg-interactive hover:outline-[*insert color here*] dark:text-black hover:dark:border-black;
```